config_tuab_eeg.py

```diff
@@ -54,8 +54,8 @@
 
 
 epochs_tmin = 0
-epochs_tmax = 10 - 1 / resample_sfreq
-rest_epochs_duration = 10. - 1 / resample_sfreq
+epochs_tmax = 10 - 1 / raw_resample_sfreq
+rest_epochs_duration = 10. - 1 / raw_resample_sfreq
```

Comment:
@MohammadJavadD can you open a PR restricted to these changes due to API updates of the pipeline? thanks